scripts/branch_rename_migration.py

```
@@ -14,19 +14,16 @@
-def run_command(command):
-    """Run a shell command and return the output."""
+def get_local_branches():
+    """Get list of local branches."""
     try:
-        result = subprocess.run(command, shell=True, capture_output=True, text=True, check=True)
-        return result.stdout.strip()
+        result = subprocess.run(["git", "branch"], shell=False, capture_output=True, text=True, check=True)
+        output = result.stdout.strip()
     except subprocess.CalledProcessError as e:
-        print(f"Error running command: {command}")
+        print(f"Error running command: git branch")
```

Flake8 (7.3.0)

[error] f-string is missing placeholders

(F541)